source/release-notes/1.1.txt

```diff
@@ -0,0 +1,44 @@
+.. _c2c-release-notes-1.1:
+
+===================================
+Release Notes for ``mongosync`` 1.1
+===================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. note::
+
+   ``mongosync`` 1.1 Released January 13, 2023
+
+
+Filtered Sync
+-------------
+
+{+c2c-product-name+} provides continuous data synchronization or a 
+one-time data migration between two MongoDB clusters. You can use
+filtered sync to control the data that the :ref:`mongosync
+<c2c-mongosync>` utility transfers between the source and destination
+clusters.
+
+For more information, see: :ref:`c2c-filtered-sync`.
+
+Unlike Sharded Topologies
+-------------------------
+
+{+c2c-product-name+} 1.1 supports sync between clusters with different
+sharding configurations.
+
+For more information, see: :ref:`c2c-api-start-sharding`.
+
+Minimum Supported Server Version
```

Comment:
We might not need to mention it here as long as we changed the source/reference/limitations.txt

Reply:
I'm going to say it's probably good to mention it here too.

Reply:
Let's mention here, good one source of truth for customers looking at 1.1.0.